web/html/jsPsychFeedback.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>jsPsych Subject Feedback</title>
+  <script src="jspsych/jspsych.js"></script>
+  <link href="jspsych/css/jspsych.css" rel="stylesheet" type="text/css">
+  <script src="src/jspsych-plugin-realtime-response.js"></script>
+  <script src="src/jsPsychWebSocket.js"></script>
+</head>
+
+<body></body>
+
+<script>
+// connect to the project server to receive events
+createWebSocket();
+
+var timeline = []
+
+// this function draws a circle with a brainstate radius
+function drawCircle(canvas, stateVal){
+  var ctx = canvas.getContext('2d');
+  var width = canvas.width
+  var height = canvas.height
+  var minDim = Math.min(width, height)
+  var brainstate = stateVal * 0.66 * minDim
```

Comment:
update this variable with an eval() command?

the function would be used like this: 

drawFunction  = drawCircle
drawFunction(parameter, stateVal)

parameter would be e.g. the string "width" then, in drawCircle we call eval(parameter + '= '+ string(stateVal)